### PR TITLE
Feature 106 refine time entry

### LINKED ISSUE #106: Refine TimeEntry
- Rename TimeEntry to WorkEvent
- Remove endTime

### PR DESCRIPTION
**Pull request by villejuutila**
Resolves #106

### CODE REVIEW

File: services/user-management-services.yaml [hunk 1]
```diff
@@ -328,13 +328,7 @@ paths:
             format: uuid
         - name: start
```
Comment (anttileppa) [inline services/user-management-services.yaml:329]: should these filter be replaced by range (after and before)?

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: services/user-management-services.yaml [hunk 2]
```diff
@@ -428,20 +417,20 @@ paths:
           schema:
             type: string
             format: uuid
-        - name: timeEntryId
+        - name: workEventId
           required: true
           in: path
-          description: time entry's id
+          description: work events id
```
Comment (anttileppa) [inline services/user-management-services.yaml:423]:
```suggestion
          description: work event's ID
```

File: services/user-management-services.yaml [hunk 3]
```diff
@@ -450,32 +439,27 @@ paths:
...
-        - TimeEntries
+        - WorkEvents
         - SpecApp
         - SpecManagementUI
       parameters:
         - name: employeeId
           required: true
           in: path
-          description: employee's id
+          description: employees id
```
Comment (anttileppa) [inline services/user-management-services.yaml:455]:
```suggestion
          description: employee's ID
```

File: services/user-management-services.yaml [hunk 4]
```diff
@@ -450,32 +439,27 @@ paths:
...
           schema:
             type: string
             format: uuid
-        - name: timeEntryId
+        - name: workEventId
           required: true
           in: path
-          description: time entry's id
+          description: work events id
```
Comment (anttileppa) [inline services/user-management-services.yaml:462]:
```suggestion
          description: work event's id
```

File: services/user-management-services.yaml [hunk 5]
```diff
@@ -516,10 +500,10 @@ paths:
...
```
Comment (anttileppa) [inline services/user-management-services.yaml:506]:
```suggestion
          description: work event's ID
```

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: services/user-management-services.yaml (hunk 1 above)
Comment (villejuutila) [reply services/user-management-services.yaml:329]: Sounds reasonable. Done.

File: services/user-management-services.yaml (hunk 2 above)
Comment (villejuutila) [reply services/user-management-services.yaml:423]: Resolved

File: services/user-management-services.yaml (hunk 3 above)
Comment (villejuutila) [reply services/user-management-services.yaml]: Resolved

File: services/user-management-services.yaml (hunk 4 above)
Comment (villejuutila) [reply services/user-management-services.yaml]: Resolved

File: services/user-management-services.yaml (hunk 5 above)
Comment (villejuutila) [reply services/user-management-services.yaml]: Resolved

**Review by anttileppa — APPROVED** [review]

**Pull request merged**